### DISCUSSION TITLE
Mention nuget package in Readme.md

### DISCUSSION THREAD

**Discussion by mfe-** [CommunityToolkit/Maui#1602] [New Feature Discussions]
## Feature Description:

As a common practice in the open-source community, it's beneficial to include direct links to required NuGet packages in the README.md file of libraries hosted on GitHub. This practice enhances the user experience by providing quick access to necessary dependencies. (see for example: https://github.com/DapperLib/Dapper?tab=readme-ov-file#features )

For our library, I propose we add a link to the [CommunityToolkit.Maui](https://www.nuget.org/packages/CommunityToolkit.Maui) NuGet package in our README.md. This will allow users to easily navigate to and download the required package, streamlining the setup process.

Benefits:

- Improves user experience by providing direct access to necessary dependencies.
- Streamlines the setup process for new users.
- Aligns with common open-source practices, enhancing the library's professionalism.

## Proposed Implementation:

Mention in the `Getting Started` the nuget package and add a link to it.

This feature request is open for further discussion and suggestions. I already started here: https://github.com/mfe-/Maui/blob/patch-1/README.md

**Comment by TheCodeTraveler** [comment]
The link to the NuGet Package is in the 3rd line of README.md

https://github.com/CommunityToolkit/Maui/blob/02896d32571af05ede9df8d6b10b32db174e15f6/README.md?plain=1#L3

![IMG_2116](https://github.com/CommunityToolkit/Maui/assets/13558917/fe955546-1e4e-4888-b2a1-2fb6f55068fe)

**Reply by mfe-** [reply]
That's a great point, having the NuGet package link in the README.md is indeed helpful. However, I believe it would also be beneficial to include it in the "Getting Started" section. This would provide new users with a clear path to the necessary dependencies right from the beginning, streamlining their initial setup process.

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
